applications/allocation_result_mapper.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from allocation.allocation_solver import AllocatedEvent
+from applications.models import ApplicationEventSchedule, ApplicationEventScheduleResult
+
+logger = logging.getLogger(__name__)
+
+
+class AllocationResultMapper(object):
+    def __init__(self, allocated_events: [AllocatedEvent]):
+        self.allocated_events = allocated_events
+
+    def to_events(self):
+        for allocated_event in self.allocated_events:
+            try:
+                application_event_schedule = ApplicationEventSchedule.objects.get(
+                    pk=allocated_event.occurrence_id
+                )
+                ApplicationEventScheduleResult.objects.update_or_create(
+                    application_event_schedule=application_event_schedule,
+                    allocated_reservation_unit_id=allocated_event.space_id,
```

Comment:
Should this reservation unit be in defaults? It can change from one unit to another.

Reply:
good point :+1:  Actually my initial thought was it would create a new one (result) but the logic does not handle deletion so I'll add it to defaults.